department-of-veterans-affairs/va.gov-cms
# VACMS-6796: vamc_system_service_audit_export - use raw html for title…

… and body fields in CSV

## Description

Closes #6796 

## Testing done

Visual

## Screenshots

![Screen Shot 2021-10-26 at 3 24 03 PM](https://user-images.githubusercontent.com/21045418/138947509-68237986-df9e-48cd-b7f3-656b3e5d5e82.png)

![Screen Shot 2021-10-26 at 3 25 35 PM](https://user-images.githubusercontent.com/21045418/138947559-a3d48272-16bd-458d-bce1-2fb0cb4f96d5.png)

## QA steps

As user with the Administrator role:

1. [ ] Visit /admin/content/audit/vamc-system-service-audit
2. [ ] Filter the results by VAMC System Description of Services contains `<h3>Care we provide at</h3>`
3. [ ] Note the number of results (in my local test I saw 17 items)
4. [ ] Scroll to the bottom and click the orange CSV button to download the CSV file
5. [ ] Open this file and verify that the number of rows match and that the Title and VAMC System Description fo Service fields contain HTML
